tests/observability/metrics/conftest.py

```diff
@@ -1100,3 +1102,15 @@ def vnic_info_from_vm_or_vmi(request, running_metric_vm):
 @pytest.fixture()
 def allocatable_nodes(nodes):
     return [node for node in nodes if node.instance.status.allocatable.memory != "0"]
+
+
+@pytest.fixture(scope="class")
+def windows_vm_for_test(namespace, unprivileged_client):
+    with create_windows10_vm(
+        dv_name="dv-for-windows",
+        namespace=namespace.name,
+        client=unprivileged_client,
+        vm_name="win-vm-for-test",
+        storage_class=StorageClassNames.CEPH_RBD_VIRTUALIZATION,
```

Comment:
Please do not use the hard-coded storage class name. 
Please use `py_config["default_storage_class"]`